pkgs/applications/gis/qgis/unwrapped-ltr.nix

```diff
@@ -29,7 +29,7 @@
 , postgresql
 , proj
 , protobuf
-, python3
+, python311
```

Comment:
I am not really sure if we need to pin python in QGIS. [QGIS is building successfully](https://hydra.nixos.org/job/nixpkgs/trunk/qgis.x86_64-linux/all) even without this PR on Hydra.